cmd/bnsd/client/client.go

```diff
@@ -280,7 +278,7 @@ func (b *BnsClient) WaitForTxEvent(tx tmtypes.Tx, evtTyp string, timeout time.Du
 	}
 
 	// make sure to unregister after the test is over
-	defer b.conn.UnsubscribeAll(ctx, uuid)
+	defer func() {_ = b.conn.UnsubscribeAll(ctx, uuid)}()
```

Comment:
I think the previous contract was much better.

Reply:
yes, but I'm not sure we can enforce error checks in one place and ignore in other unless we use `//nolint`
I have seen how errcheck allows to find stuff that looks like bugs and missing error checks, so I'm kind of lost.